### PR TITLE
Initial draft for the polkadot standard proposal

### PR DESCRIPTION
**Pull request by Noc2**
I just created a quick first draft.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,24 @@
...
+1. Draft -> 2. Call for Feedback -> 3. Accepted -> 4. Integrated
+```
+1. **Draft:** A first draft, which is merged into this repo and actively improved together with the community. 
+2. **Call for Feedback:** The PSP will be shared on different channels for additional feedback for at least 2 weeks. The result of this step is either an acceptance of the standard (->Accepted) or the rejection (->Draft). 
+3. **Accepted:** Any further changes are unlikely, and developers can start integrating the PSP. Once a PSP is accepted, a reference implementation needs to be created.    
+4. **Integrated:** The PSP is actively used and a reference implementation exists. . 
+
+## Contributing PSPs
+
+Before you start writing a formal PSP, you should discuss an idea in the various community channels. A PSP should provide the motivation as well as a technical specification for the feature.
```
Comment (laboon) [inline README.md]: Perhaps we could add some links to the various community channels here (Riot, forum, etc.)

File: psp-template.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Title
+
+* **PSP Number:** [To be assigned]
+* **Authors:** [Name and Email or GitHub Username]
+* **Status:** [Draft, Call for Feedback, Accepted, Integrated]
+* **Created:** [yyyy-mm-dd]
+* **Reference Implementation** [Link to a first reference implementation]
+
+## Summary
```
Comment (laboon) [inline psp-template.md]: We may want to go through making the ERC-20 equivalent and use it as a reference, then link to it here as a "good example".

File: README.md (hunk 1 above)
Comment (keorn) [reply README.md]: Is this good: https://polkadot.network/community/ ? Or do we have a page on wiki with more technical selection?

Comment (Noc2) [reply README.md]: I added a link to the community website. Should be sufficient for now.

**Pull request merged**